src/platform_impl/macos/window.rs

```diff
@@ -344,6 +356,13 @@ impl WinitWindow {
                     }
                 }
 
+                if let Parent::ChildOf(parent) = pl_attrs.parent {
+                    #[allow(clippy::let_unit_value)]
+                    unsafe {
+                        let () = msg_send![parent as *mut Object, addChildWindow: &*this, ordered: NSWindowOrderingMode::NSWindowAbove];
```

Comment:
This could be added as an `unsafe` method on `NSWindow` instead, which would give me more confidence in the safety than this inline usage does